src/vivarium_gates_child_iv_iron/constants/paths.py

```diff
@@ -5,11 +5,11 @@
 
 BASE_DIR = Path(vivarium_gates_child_iv_iron.__file__).resolve().parent
 
-ARTIFACT_ROOT = Path(f"/share/costeffectiveness/artifacts/{metadata.PROJECT_NAME}/")
+ARTIFACT_ROOT = BASE_DIR / "artifacts"
```

Comment:
nit: inconsistent quote types. I'm surprised black didn't pick this up

Reply:
We don't make substantive changes in archival.